modules/gsp-cluster/nlb.tf

```diff
@@ -0,0 +1,83 @@
+resource "aws_security_group" "ingress-nlb" {
+  count = "${var.enable_nlb == 1 ? 1 : 0 }"
+
+  name        = "${var.cluster_name}-ingress-nlb"
+  description = "${var.cluster_name} ingress (NLB) security group"
```

Comment:
Err... NLBs don't have security groups. Broadly, the client IP is magically forwarded through to the backend, so it's up to whether the backend SG accepts the client IP.